db/fixtures/paths/full_stack_rails/courses/intermediate_html_css.rb

```diff
@@ -49,6 +47,21 @@
   )
 end
 
+# +++++++++++++++++++++++++++++++++++++++
+# # SECTION - Forms
+# # +++++++++++++++++++++++++++++++++++++++
+course.add_section do |section|
+  section.title = 'Forms'
+  section.description = "Forms are a crucial building block for getting input from your users. In this section you'll learn some of the basics you'll need to create a functioning form."
+  section.identifier_uuid = 'b72a5edc-f513-11ec-b939-0242ac120002'
+
+  section.add_lessons(
+    html_and_css_lessons.fetch('Form Basics'),
+    html_and_css_lessons.fetch('Form Validation'),
+    html_and_css_lessons.fetch('Sign-up Form'),
```

Comment:
Can the `html_and_css_lessons.fetch('Sign-up Form')` line be removed from the Intermediate CSS Concepts section above in this file please?

Reply:
arghh. sorry, I should've double checked! I removed it as requested and have pushed the changes. Happy to help Kevin!